internal/pkg/manifest/svc.go

```diff
@@ -206,12 +209,18 @@ func (a *AdvancedCount) hasAutoscaling() bool {
 func (a *AdvancedCount) IsValid() error {
```

Comment:
i wonder if we still need this function given that we have validated everything?

Reply:
That's the next step when i consolidate every validation else where!